components/_util/wave.tsx

```diff
@@ -195,7 +199,19 @@ export default class Wave extends React.Component<{ insertExtraNode?: boolean }>
     const { children } = this.props;
     this.csp = csp;
 
-    return children;
+    if (!React.isValidElement(children)) return children;
+
+    let ref: React.Ref<any> = this.containerRef;
+    if (supportRef(children)) {
+      ref = setAndForwardRef({
+        getForwardedRef: () => (children as any).ref,
+        setLocalRef: _ref => {
+          (this.containerRef as any).current = _ref;
+        },
+      });
+    }
+
+    return cloneElement(children, { ref });
```

Comment:
Use `composeRef` directly instead of `setAndForwardRef`:

```tsx
if (supportRef(children)) {
  return cloneElement(children, {
    ref: composeRef(this.containerRef, children.ref),
  });
}
```

Reply:
PS: `composeRef` is in `rc-util/lib/ref`